lib/MusicBrainz/Server/Email.pm

```diff
@@ -446,8 +409,25 @@ sub send_email_verification
 {
     my ($self, %opts) = @_;
 
-    my $email = $self->_create_email_verification_email(%opts);
-    return $self->_send_email($email);
+    my $verification_link = $opts{verification_link};
+    my $ip = $opts{ip};
+    my $user_name = $opts{editor}->name;
+
+    my $body = {
+        template_id => 'verify-email',
+        to          => $opts{email},
+        from        => $EMAIL_NOREPLY_ADDRESS,
+        reply_to    => $EMAIL_NOREPLY_ADDRESS,
+        # TODO: send the user's language preference here. 
+        message_id  => _message_id('verify-email-%s', generate_gid()),
+        params      => {
+            to_name          => $user_name,
+            verification_url => ${ $verification_link },
+            # TODO: Do we include the IP address, like with the previous email?
```

Comment:
I can't remember, did we previously discuss removing it? I'm fine with dropping it, anyway.